editor/src/clj/editor/gl/pass.clj

```diff
@@ -41,9 +41,9 @@
 (make-passes
   ; name selection model-transform depth-clipping
   background            false false false
-  infinity-grid         false true  false
   opaque                false true  true
   transparent           false true  true
+  infinity-grid         false true  false
```

Comment:
I moved the grid pass above the scene items. I think it makes sense to render the grid on top of your scene. A substantial amount of objects may occlude the grid, if we render it on the background. Now that the toggle is a top level toolbar control, it is easy to deactivate it, and we also added a default shortcut. We can make this configurable if we receive complains, but I don't think we should do that yet.